Sources/scipio/scipio.docc/build-pipeline.md

```diff
@@ -246,10 +248,37 @@ You can also implement your custom cache storage by implementing `CacheStorage`
 
 There are two cache actors `consumer` and `producer`.
 
-You can specify it by a second argument of `.storage` cache mode.
+You can specify it by `Runner.Options.CachePolicy.actors`.
 
 `consumer` is an actor who can fetch cache from the cache storage.
 
 `producer` is an actor who attempt to save cache to the cache storage.
 
 When build artifacts are built, then it try to save them.
+
+### Use multiple cache policies at the same time
+
+You can also use multiple cache policies, which accepts multiple cache storages with different sets of cache actors:
+
+```swift
+import ScipioS3Storage
+
+let s3Storage: some CacheStorage = ScipioS3Storage.S3Storage(config: ...)
+let localCacheStorage: some CacheStorage = LocalCacheStorage()
```

Comment:
LocalDiskCacheStorage is public type and can be initialized with custom cache directory.

Reply:
Yes!

Because we want to restrict their actors. e.g. for `ProjectCache`, users can only use `[.producer]` combination.

To conceal the storage constructed, we can prevent mistakes in usage.